<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="mainPage">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>380</y>
       <width>91</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="selectMenu">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>开始游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="aboutButton">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>退出游戏</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="levelPage">
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>329</y>
       <width>254</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="levelButton1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="levelButton2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="levelButton4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="levelButton3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="backButton1">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>550</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settingsPage">
    <widget class="QPushButton" name="backButton2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>500</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>310</y>
       <width>361</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <widget class="QLabel" name="volumnValue">
        <property name="text">
         <string>50</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSlider" name="volumnHorizontalSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="selectFileDirectory">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>BGM 音量大小</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>存档位置</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="fileDirectory">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="gamePage">
    <widget class="QLabel" name="robotLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>330</y>
       <width>69</width>
       <height>65</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>470</y>
       <width>451</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="vacantSpaces">
      <item>
       <widget class="QLabel" name="vacant0">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vacant1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vacant4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vacant3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vacant2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vacant5">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="announcementLabel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>69</width>
       <height>65</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>261</width>
       <height>211</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>259</width>
        <height>209</height>
       </rect>
      </property>
      <widget class="QTextEdit" name="cmdTextEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>241</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="confirmNextStepButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>370</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton3">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>570</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentStepLabel">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>370</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="inputTextLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>69</width>
       <height>411</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="outputTextLabel">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>67</y>
       <width>69</width>
       <height>381</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
